client/layout/multi-sites-dashboard/item-view/index.tsx

```diff
@@ -86,16 +86,18 @@ export default function ItemPreviewPane( {
 	const shouldHideNav = ( hideNavIfSingleTab && featureTabs.length <= 1 ) || isMobileApp;
 
 	return (
-		<div className={ clsx( 'item-preview__pane', className ) }>
-			<ItemPreviewPaneHeader
-				closeItemPreviewPane={ closeItemPreviewPane }
+		<div className={ clsx( 'multi-sites-dashboard-item-view', className ) }>
```

Comment:
I wonder if naming the component `multi-sites-dashboard` will be too restrictive by presuming that it will be used for sites only. WDYT?

Reply:
Hmm, you're right! I'll address this in a separate PR. I think `hosting-dashboard-layout` might better align with the goals of the redesign. p9Jlb4-fpU-p2#comment-14878

Reply:
> hosting-dashboard-layout 

It sounds perfect!